packages/connect-webextension/src/proxy/index.ts

```diff
@@ -0,0 +1,134 @@
+import EventEmitter from 'events';
+
+// NOTE: @trezor/connect part is intentionally not imported from the index due to NormalReplacementPlugin
+// in packages/suite-build/configs/web.webpack.config.ts
+import {
+    ERRORS,
+    IFRAME,
+    POPUP,
+    createErrorMessage,
+    ConnectSettings,
+    Manifest,
+    CallMethod,
+} from '@trezor/connect/lib/exports';
+import { factory } from '@trezor/connect/lib/factory';
+
+import { parseConnectSettings } from '../connectSettings';
+import { WindowServiceWorkerChannel } from '../channels/window-serviceworker';
+
+const eventEmitter = new EventEmitter();
+let _settings = parseConnectSettings();
+let _channel: any;
+
+const manifest = (data: Manifest) => {
+    _settings = parseConnectSettings({
```

Comment:
should this happen here? isn't it duplicated with logic in service worker? maybe settings shouldn't be handled here at all?

Reply:
All the logic related to settings has been removed from there https://github.com/trezor/trezor-suite/pull/10751/commits/c294594cbcbc44bc391611dccb010090434d9de3